zuul.d/projects.yaml

```diff
@@ -13,6 +13,10 @@
     name: ansible-collections/community.cassandra
     default-branch: master
 
+- project:
+    name: ansible-collections/community.lbry
+    default-branch: main
```

Comment:
This is necessary if the default-branch is not main, it's otherwise covered by the regex at the beginning of the file.

In all likelihood it should "just work" once we merge the first PR (https://github.com/ansible/zuul-config/pull/311) and this PR is no longer necessary,

Reply:
Cool, thanks for clarifying!